kubernetes/phpmyadmin.yml.erb

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: phpmyadmin-service
+spec:
+  selector:
+    app: phpmyadmin
+  ports:
+    - port: 80
+      targetPort: 8000
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: phpmyadmin-deployment
+  labels:
+    app: phpmyadmin
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: phpmyadmin
+  template:
+    metadata:
+      labels:
+        app: phpmyadmin
+    spec:
+      containers:
+        - name: phpmyadmin-deploy
+          image: "docker.ocf.berkeley.edu/pma:<%= version %>"
+          resources:
+            limits:
+              memory: 1Gi
+              cpu: 1000m
+          ports:
+            - containerPort: 8000
+      dnsPolicy: ClusterFirst
+      dnsConfig:
+        searches:
+          - "ocf.berkeley.edu"
```

Comment:
Since the "standard" Kubernetes templates are getting quite large with these kinds of additions, I think it could be worth looking into [template partials](https://github.com/Shopify/kubernetes-deploy#using-partials) to cut down on this inevitable repetition between services